--- origsrc/fossil/src/main.c	2008-09-28 04:02:29.000000000 -0500
+++ src/fossil/src/main.c	2008-09-28 04:15:23.438504400 -0500
@@ -709,7 +709,7 @@
   /* Unix implementation */
   if( g.argv[1][0]=='u' ){
 #if !defined(__DARWIN__) && !defined(__APPLE__)
-    zBrowser = db_get("web-browser", "firefox");
+    zBrowser = db_get("web-browser", "xdg-open");
 #else
     zBrowser = db_get("web-browser", "open");
 #endif
