--- origsrc/fossil-src-20091220213451/src/main.c	2009-12-21 08:33:29.000000000 -0600
+++ src/fossil-src-20091220213451/src/main.c	2009-12-22 11:28:21.820138600 -0600
@@ -806,7 +806,7 @@ void cmd_webserver(void){
 #if !defined(__DARWIN__) && !defined(__APPLE__)
     zBrowser = db_get("web-browser", 0);
     if( zBrowser==0 ){
-      static char *azBrowserProg[] = { "xdg-open", "gnome-open", "firefox" };
+      static char *azBrowserProg[] = { "xdg-open", "gvfs-open", "kde-open", "epiphany", "midori", "arora", "konqueror", "cygstart" };
       int i;
       zBrowser = "echo";
       for(i=0; i<sizeof(azBrowserProg)/sizeof(azBrowserProg[0]); i++){
